packages/crayons-extended/custom-objects/src/components/platform-table/platform-table.scss

```diff
@@ -16,7 +16,8 @@
   border: 1px solid
     var(--color-boundary-border-mildest, rgba(207, 215, 223, 0.4));
   box-sizing: border-box;
-  border-radius: 4px;
+  border-radius: 0 0 var(--border-radius-medium, 4px)
```

Comment:
Please update the dew variables and fallback values for all below 4 values of the border radius . 
    border-top-left-radius: 4px;
    border-top-right-radius: 4px;
    border-bottom-right-radius: 4px;
    border-bottom-left-radius: 4px;